crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -226,18 +241,100 @@ mod tests {
 
         // enough balance; should pass
         let expected_deposit_fee = transfer_fee
-            + get_deposit_byte_len(&Deposit {
+            + calculate_base_deposit_fee(&Deposit {
                 bridge_address,
                 rollup_id,
                 amount: 100,
                 asset: asset.clone(),
                 destination_chain_address: "someaddress".to_string(),
                 source_transaction_id: transaction_id,
                 source_action_index: 0,
-            }) * 2;
+            })
+            .unwrap()
+                * 2;
         state
             .put_account_balance(from_address, &asset, 100 + expected_deposit_fee)
             .unwrap();
         bridge_lock.check_and_execute(&mut state).await.unwrap();
     }
+
+    #[test]
+    fn calculated_base_deposit_fee_matches_expected_value() {
+        assert_correct_base_deposit_fee(&Deposit {
+            amount: u128::MAX,
+            source_action_index: u64::MAX,
+            ..default_deposit()
+        });
+        assert_correct_base_deposit_fee(&Deposit {
+            asset: "test_asset".parse().unwrap(),
+            ..default_deposit()
+        });
+        assert_correct_base_deposit_fee(&Deposit {
+            destination_chain_address: "someaddresslonger".to_string(),
+            ..default_deposit()
+        });
+
+        // Ensure calculated length is as expected with absurd string
+        // lengths (have tested up to 99999999, but this makes testing very slow)
+        let absurd_string: String = ['a'; u16::MAX as usize].iter().collect();
+        assert_correct_base_deposit_fee(&Deposit {
+            asset: absurd_string.parse().unwrap(),
+            ..default_deposit()
+        });
+        assert_correct_base_deposit_fee(&Deposit {
+            destination_chain_address: absurd_string,
+            ..default_deposit()
+        });
+    }
+
+    #[track_caller]
+    #[allow(clippy::arithmetic_side_effects)] // allow: test will never overflow u128
+    fn assert_correct_base_deposit_fee(deposit: &Deposit) {
+        let calculated_len = calculate_base_deposit_fee(deposit).unwrap();
+        let expected_len = DEPOSIT_BASE_FEE
```

Comment:
This just copies what's inside `fn calculate_base_deposit_fee`. I think you should replace this with a fixed number instead, because all of this is deterministic.